lib/mods/replace-window-functions.js

```diff
@@ -0,0 +1,57 @@
+'use strict';
+
+const nodeTypes = require('ts-morph').TypeGuards;
+const {updateImports} = require('../imports.js');
+const {
+  formatReplacement,
+  getOriginal,
+} = require('../originals-helper.js');
+
+/**
+ * @param {*} node - the node to examine
```

Comment:
What type is this? It'd be good to do a first pass before sending things for review to catch this sort of thing.